src/client/modules/my/dataUpload/dataUpload.ts

```diff
@@ -0,0 +1,62 @@
+import { LightningElement } from 'lwc';
+
+export default class DataUploader extends LightningElement {
+
+    /**
+     * Retrieves the data from the input fields and makes a query to upload it to the database api.
+     */
+    gatherDataAndSend() {
+
+        let playerData = []; //data for each player input
+
+        //get entered player data
+        for(let x = 0; x < 6; x++) {
+            playerData.push({
+                "playerName": this.getValueFromInput("playerName" + (x+1).toString()),
+                "victoryPoints": parseInt(this.getValueFromInput("victoryPoints" + (x+1).toString()), 10)
+            });
+        }
+
+        //data for post
+        let data = {
+            "gameId": this.getValueFromInput("gameId"),
+            "playerData": playerData
+        };
+
+        console.log(data);
+        console.log(JSON.stringify(data));
```

Comment:
Fun console hack:
`console.log("data", data)`
will make the data object interactable (open/closable)